examples/custom-bucket-policy-inventory/main.tf

```diff
@@ -0,0 +1,58 @@
+#
+# Custom Bucket Policy
+#
+data "aws_caller_identity" "current" {}
+
+data "aws_iam_policy_document" "custom_bucket_policy" {
```

Comment:
Since the focus of this example is primarily on the bucket inventory rather than the bucket policy, should we remove the custom bucket policy from this example and rename the folder to `examples/bucket-inventory` (or something similar?